### ISSUE TITLE
Agenda for Internship Stand Up

### ISSUE THREAD

**Issue by Zytko** [hackforla/internship#9]
### Overview
This issue tracks the agenda for the product management community of practice and weekly roll call for projects.

Please check in on the [Roll Call Spreadsheet](https://docs.google.com/spreadsheets/d/11KC6P07dr_DEIq5lGZ_vqyakq-rfOEBSTqqduFLiVbs/edit#gid=0)

### Weekly Agenda
[06/16](https://github.com/hackforla/internship/issues/9#issuecomment-862590492)
[06/17](https://github.com/hackforla/internship/issues/9#issuecomment-862621611)
[06/18](https://github.com/hackforla/internship/issues/9#issuecomment-863390444)
[6/21](https://github.com/hackforla/internship/issues/9#issuecomment-865106974)
[6/22](https://github.com/hackforla/internship/issues/9#issuecomment-866115417)
[6/23](https://github.com/hackforla/internship/issues/9#issuecomment-866983607)
[6/25](https://github.com/hackforla/internship/issues/9#issuecomment-868676527)
[6/28](https://github.com/hackforla/internship/issues/9#issuecomment-869818433)
[6/29](https://github.com/hackforla/internship/issues/9#issuecomment-870737576)
[6/30](https://github.com/hackforla/internship/issues/9#issuecomment-870912639)
[7/1](https://github.com/hackforla/internship/issues/9#issuecomment-872374571)
[7/2](https://github.com/hackforla/internship/issues/9#issuecomment-873110535)
[7/5](https://github.com/hackforla/internship/issues/9#issuecomment-874225946)
[7/6](https://github.com/hackforla/internship/issues/9#issuecomment-874916941)
[7/7](https://github.com/hackforla/internship/issues/9#issuecomment-875733443)
[7/9](https://github.com/hackforla/internship/issues/9#issuecomment-877296252)
[7/12](https://github.com/hackforla/internship/issues/9#issuecomment-878413282)
[7/13](https://github.com/hackforla/internship/issues/9#issuecomment-878809873)
[7/14](https://github.com/hackforla/internship/issues/9#issuecomment-880035145)
[7/15](https://github.com/hackforla/internship/issues/9#issuecomment-880828746)
[7/16](https://github.com/hackforla/internship/issues/9#issuecomment-881561613)
[7/19](https://github.com/hackforla/internship/issues/9#issuecomment-882675278)
[7/20](https://github.com/hackforla/internship/issues/9#issuecomment-883513015)
[7/21](https://github.com/hackforla/internship/issues/9#issuecomment-884325331)
[7/22](https://github.com/hackforla/internship/issues/9#issuecomment-884999556)
[7/23](https://github.com/hackforla/internship/issues/9#issuecomment-884999725)
[7/26](https://github.com/hackforla/internship/issues/9#issuecomment-885000553)
[7/27](https://github.com/hackforla/internship/issues/9#issuecomment-886863956)
[7/28](https://github.com/hackforla/internship/issues/9#issuecomment-888434870)
[7/29](https://github.com/hackforla/internship/issues/9#issuecomment-889277693)
[08/06](https://github.com/hackforla/internship/issues/9#issuecomment-894375120)
[08/09](https://github.com/hackforla/internship/issues/9#issuecomment-895354660)
[08/11](https://github.com/hackforla/internship/issues/9#issuecomment-896959506)
[08/12](https://github.com/hackforla/internship/issues/9#issuecomment-897764945) - transcript and recording of standup

**Comment by Zytko** [comment]
### Overview
Agenda for 6/17/2021

### Action Items

**Comment by ExperimentsInHonesty** [comment]
Agenda for 6/17/2021
- Confirm that I have a 1-1 setup with each intern
- Add a check to the roll call spreadsheet
- Ask for:
   - Progress
   - Blockers
   - Availability
   - ETA

**Comment by Zytko** [comment]
Agenda for 6/18/2021

**Comment by Zytko** [comment]
Agenda for 6/21/2021

Roll Call
Sakari
Progress: 
Blocker: 
Michelle
Progress: 
Blocker: 
ETA: 
Mateo
Progress: 
Blocker:
ETA: 
Sophia
Progress:
Blockers:

**Comment by Zytko** [comment]
Agenda for 6/22/2021
Roll Call:
Sakari
- Progress: Finished first and second issue
- Blocker: Figuring out how to do two PR's at the same time
Sophia
- Progress: Began working on a guide
- Blocker: None (Need to meet with Danielle)
Mateo
- Progress: Set up GitHub, completed first issue
- Blocker: Need better understanding of Docker
Michelle---
Progress:---
Blockers:---

**Comment by Zytko** [comment]
Agenda for 6/22/2021
Roll Call:

Sakari
Progress: Made changes after review, worked on Docker issue, and reviewed 5+ issues
Blocker: Waiting for reviews on code

Sophia
Progress: Finished replicating design for team meeting website
Blocker: None

Mateo
Progress: Completed second issue and pull request and worked on a guide
Blocker: No mentions of secrets or credentials in read me's

Michelle---
Progress:---
Blockers:---

**Comment by ExperimentsInHonesty** [comment]
Agenda for 6/25/21

Sakari
Progress: Updated the Docker issue, reviews, got response to pr (medium) will implement changes
Blocker: needed to know how to display projects with status other than active.

Sophia
Progress: submitted good 2nd issue PR waiting for update 
Blocker: none

**Comment by Zytko** [comment]
Agenda for 6/28

Sakari
Progress: New issue (standardizing programs page on mobile), couple of reviews, docker issue back in prioritized backlog
Blocker: Typography issue still in discussion

Sophia
Progress: Finished looking for website pictures, finished second issue
Blocker: Need to meet with Danielle to make progress on design system, needs an appropriate sized medium developer issue

Jason
Progress: Made website meeting on Sunday, set up developer environment
Blocker: None

Mateo
Progress: Finished good second issue, going to start good second issue
Blocker: Needs to find appropriate medium sized issue

Michelle---

**Comment by Zytko** [comment]
Agenda for 6/29

Sakari
Progress: Fixed typography issue, done with standardized programs page for mobile, did multiple reviews
Blocker: None

Sophia
Progress: Finished two of three bulletins for issue, updated good second issue, reviewed by Ava on good second issue, help Jason set up dev environment
Blocker: Missing one file and it is not clear from the site architecture where it should be

Jason
Progress: Set up dev environment
Blocker: Git cannot download because not enough storage

**Comment by Zytko** [comment]
Agenda for 6/29

Make sure to reschedule one on one with Sakari

Roll Call:
Sakari
Progress:
Blockers

Sophia
Progress:
Blockers:

Mateo
Progress:
Blockers:

Jason
Progress:
Blockers:

**Comment by Zytko** [comment]
Agenda for 7/1

Sakari
Progress: Resolved conflict between his and Abel's issue
Blocker: Sick

Sophia
Progress: Out of pull request limbo, fixed third issue
Blockers: Waiting for PR

Jason
Progress: Got Docker to work! Continuing Dev Environment setup
Blockers: None

**Comment by ExperimentsInHonesty** [comment]
Agenda for 7/2/2021

Roll Call:

Sakari
Progress: Reviews and LinkedIn Profile
Blocker: need feedback on potential solution for issue

Jason
Progress: will finish setting up dev environment today
Blockers: busy

**Comment by Zak234** [comment]
Agenda for 7/5/2021

Roll Call:

Sakari
Progress: Helped Jack with on boarding 
Blocker: Stuck on code changes (meeting with Josh about it)

Mateo
Progress: Finished second issue, working on medium sized issue
Blockers:

**Comment by ExperimentsInHonesty** [comment]
Agenda for 7/6/2021

Roll Call:

Sakari
Progress: Met with Josh to talk though the changes, redid the pull request, linkedin profile, pr reviews
Blocker: cant see the command line code for running their code.

Sophia
Progress: submitted pr, received comments about mobile changes, start a design issue
Blockers:  waiting for help today, posted in engineering rather than hfla-site

Jason
Progress: Got Dev environment setup, ready to take on first issue
Blockers: none

**Comment by Zytko** [comment]
7/7

Roll Call:

Sakari
Progress: Worked on Docker and Template guide, posted in Slack channel about bug
Blocker: Waiting for PR to be merged

Jason
Progress: None
Blocker: None

Sophia
Progress: Working on design issue
Blocker: Third issue to be merged

Mateo
Progress: Still working on medium sized issue Solved blocker installing Ruby for Jekyll
Blocker: None

**Comment by Zytko** [comment]
7/9

Roll call

Jason
Progress: Met with Glen to discuss first issue, finished first issue, working on Linkedin
Blocker: None

Sophia
Progress: PR approved, to be merged soon, formatting issue solved, workin on Linkedin
Blocker: None

Sakari
Progress: Met with Ava to fix VMS issue, continued to work on Docker guide
Blocker: Had to re-request PR

**Comment by ExperimentsInHonesty** [comment]
7/13

Sakari
Progress: On a new large issue
Blockers: none

Sophia
Progress: working on second medium for mobile on about
Blockers: waiting for ed to provide text on citizen engagement

Jason
Progress: going to start good second issue
Blockers: none

**Comment by JasonUranta** [comment]
Jason
Progress: made progress on 2nd issue
Blockers: none

**Comment by ExperimentsInHonesty** [comment]
7/14/21

Sakari
Progress: Large issue
Blockers: lack of knowledge

Sophia:
Progress:  Finished the Citizen Engagement issue, posted in the hfla-design channel 
Blockers: issue being merged

Jason: 
Progress:
Blockers: took day off for birthday

Mateo:
Progress: Almost done with med issue
Blocker: need help figuring out why the image wont show on credits

**Comment by ExperimentsInHonesty** [comment]
7/15

Sakari
Progress: Large Issue
Blockers: not clear how to check back end work to see if its performing as expected

Sophia
Progress: Making changes based off design feedback Citizen Engagement
Blockers: waiting for pr review

Jason
Progress: Ready to submit 2nd PR looking for confirmation of method
Blockers:

**Comment by ExperimentsInHonesty** [comment]
7/16/21

Sakari
Progress: Met with Ava for test coaching
Blockers: Figuring it out

Sophia
Progress:  Will finish the changes requested and communicate with Danielle
Blockers: no one has reviewed pull request

Mateo
Progress: Finished getting PR reviewed by danielle and got signoff
Blockers: Need a review from other developers.

**Comment by ExperimentsInHonesty** [comment]
7/19

Sakari
Progress: Large Issue, contributed to making a new issue for the credits file
Blockers:

Jason
Progress: Finished Second Issue
Blockers: needs to respond to the comment and get the merge to be able to move on

Mateo
Progress: Need to fix tab spacing
Blockers: none

**Comment by ExperimentsInHonesty** [comment]
7/20

Sakari
Progress: Large issue
Blockers: Will seek help from meeting tonight

Jason: Self Assigned issue medium
Blocker: availability

**Comment by ExperimentsInHonesty** [comment]
7/21/21

Sakari:
Progress: Started talking to potential mentors
Blockers: Need to communicate with them about their time zones

Jason:
Progress: Working on medium issue
Blockers: none

Mateo
Progress: Ready to work on Large issue
Blockers: none

**Comment by ExperimentsInHonesty** [comment]
7/22/21

**Comment by ExperimentsInHonesty** [comment]
7/23/21

**Comment by ExperimentsInHonesty** [comment]
7/26/21

Sakari

Jason

Mateo

Sophia
Ask Sophia to fill out th#14

**Comment by ExperimentsInHonesty** [comment]
7/27/21

Sakari
Ask him for a link to Resume Content Tool spreadsheet (add members of internship team to it, so that they can view)

**Comment by ExperimentsInHonesty** [comment]
7/28

Sakari
Progress: Linkedin Profile, Large Issue, Attended Ava's breakout on GitHub action
Blockers: 

Mateo: Medium Issue
Blockers: Availability

Jason: About draft and linkedin 
Blockers:

**Comment by ExperimentsInHonesty** [comment]
7/29

Sakari
Progress: working on large issue
Blockers: a little stuck, meeting with Community of Practice mentor

Sophia
Progress: researching and testing medium front end issue
Blockers: making the mobile/tablet behave properly

Jason
Progress: linked in profile
Blockers: none

Mateo
Progress
Blockers

Jack
Progress
Blockers

**Comment by ExperimentsInHonesty** [comment]
7/30/21

Sakari:
Progress: large issue met with Jonathan Shin for mentorship
Blockers: Internet connection

Sophia
Progress: finished tablet version, trying to figure out 
Blockers: some issue with git pull

Jason
Progress: LinkedIn
Blockers: Availability

**Comment by ExperimentsInHonesty** [comment]
Sakari:
Progress: large issue 
Blockers: trying to meet up with Jonathan Shin and Frank Stepanski

Sophia
Progress: got a determination at the meeting of what to do with Table view and made progress with auto layout
Blockers: 

Jason
Progress: Going to make changes requested and review an issue
Blockers: Availability

Mateo
progress: made good progress on medium issue, test with Sakari
blockers: no blockers

**Comment by ExperimentsInHonesty** [comment]
8/3/21

When to seek help on their work because and when to keep researching 
How to keep other team members availability from acting a blocker to their work 
Documenting your work for your peers/people you report to 
Finding mentors outside of your department
Presenting your education and work on your LinkedIn profile

Sakari
Progress: 
Blockers:

Sophia
Progress: 
Blockers:

Jason
Progress: 
Blockers:

Mateo
Progress: 
Blockers:

Jack
Progress: 
Blockers:

**Comment by ExperimentsInHonesty** [comment]
08/07/21

Sakari
Progress: Large issue worked with Jonathan Shin, and Ava and determined Feasibility was not worth the lift at this time.  Will update wiki and probably take on medium or large GitHub actions issue
Blockers: none
Throw that on a wiki page
Title of page Problem-Solution Deferred Documentation
Problem statement:
Potential Solution:
Feasibility Determination:

Jason
Progress: Reviewing pull request, 
Blockers: error messages

Mateo
Progress: will be making a donate issue
Blockers: waiting for a pr review

**Comment by sniux** [comment]
8/5/21

Sakari
Progress: made a [wiki page](https://github.com/hackforla/website/wiki/Problem-Solution-Deferred-Documentation
) ; will be taking on large issue
Blockers: none

Sophia
Progress: waiting on comments to finish the dev issue, almost done with design issue
Blockers: waiting on design team

Jason
Progress: finished reviewing pull request, worked on LinkedIn profile
Blockers: moved the 1-on-1 meet

**Comment by ExperimentsInHonesty** [comment]
8/6/21

Sakari
Progress: New large issue, more research
Blockers: none
Continue internship until 27th

Sophia
Progress: Finished design issue and its in the review column, Requested re review on PR after making requested changes
Blockers: design issue waiting on Kristine visual review and PR waiting on re-review from Josh and Adam
Reduce internship to 1-on-1s until August 26th

Jason
Progress: Linkedin profile/spreadsheet, will start on the https://github.com/hackforla/website/issues/2008 and estimate to potentially be done by Tuesday.
Blockers: Availability
1-on1- on Friday evening - evaluate time required to finish internship with goals met.

Mateo
Progress: Met with Ava on Medium issue (got merged, pr review completed)
Blockers:
1-on1- on Monday morning - evaluate time required to finish internship with goals met.

**Comment by ExperimentsInHonesty** [comment]
8/9/21

Sakari
Progress: New Large issue - github comment api
Blockers: 
check out Kian's repo https://github.com/100Automations/true-github-contributors

Jason
Progress: Finished issue, now working on linkedin profile
Blockers: waiting for review

Jack
Progress: Working on a github actions issue
Blockers:  where the file was and how to edit (resolved)

**Comment by ExperimentsInHonesty** [comment]
8/11/21

Sakari
Progress: 
Blockers:

Jason
Progress: Received request for PR changes, making them, and working on linkedin spreadsheet
Blockers: none

Jack
Progress: received merge conflict and reached out to Ava and she provided some info about merge conflicts.  Made changes.
Blockers: none

**Comment by ExperimentsInHonesty** [comment]
8/12/21

Sakari
Progress: Working on the large issue, have been talking to Ava and will do some pair programing.  
Blockers: None

Jason
Progress: Going to reply to feedback on the PR, Got some linkedin done and will continue to work on it in prep for our meeting later today
Blockers: Availability

**Comment by ExperimentsInHonesty** [comment]
8/13/21

Jason
Progress: none, work through issue to be ready for Sunday
Blockers: Availability

Sakari
Progress: Pair programing with Ava, attended backend meeting, still working with issue
Blockers: none

**Comment by ExperimentsInHonesty** [comment]
Confirm that Sakari has accepted the Monday invite for interview at 9:10

**Comment by ExperimentsInHonesty** [comment]
8/19/21

Jason
Progress: Linkedin Spreadsheet almost done, worked with Felicia and Bonnie to figure out size of standard Git install
Blockers: General Availability

**Comment by ExperimentsInHonesty** [comment]
8/20/21

Jack
Progress: Linked in activity log update, pr review
Blockers: unsure of what goes in each field in the activity log

**Comment by ExperimentsInHonesty** [comment]
8/23/21

Jason
Progress: done with linkedin profile, 
Blockers: having issue with branching for contributing file

sakari: will work on issue this week, now that its been descoped.

**Comment by ExperimentsInHonesty** [comment]
8/25/21

Jack
Progress: started large issue https://github.com/hackforla/website/issues/1407
Blockers: the last three bullets are confusing

**Comment by ExperimentsInHonesty** [comment]
We finished all the standups.
